index.js

```diff
@@ -141,6 +144,12 @@ File.prototype.inspect = function() {
   return '<File ' + inspect.join(' ') + '>';
 };
 
+File.builtInFields = ['_contents', 'contents', 'stat', 'history', 'path', 'base', 'cwd'];
```

Comment:
The only still comfy alternative I can think of would be to just inline it into the `.isCustomProp()` method:

```js
File.isCustomProp = function(key) {
  return ['_contents', 'contents', 'stat', 'history', 'path', 'base', 'cwd'].indexOf(key) === -1;
};
```

And anyone that extends `Vinyl` will just extend the method when needed:

```js
class Vinyl2 extends Vinyl {
  static isCustomProp(key) {
    return super.isCustomProp(key) && ['my', 'own', 'props'].indexOf(key) === -1;
  }
}
```